--- bogofilter-0.92.2/src/configfile.c	2004-03-20 15:54:39.000000000 +0000
+++ bogofilter-0.92.2.new/src/configfile.c	2004-07-27 08:10:56.495995144 +0000
@@ -123,7 +123,13 @@
     int lineno = 0;
     FILE *fp;
     char *filename;
+    char *home_etc = getenv("HOME_ETC");
 
+    if (home_etc && fname[0] == '~') {
+        char buffer[1024];
+	snprintf(buffer, 1024, "%s%s", home_etc, fname+1);
+	filename=xstrdup(buffer);
+    } else
     filename = tildeexpand(fname, tilde_expand);
 
     fp = fopen(filename, "r");
--- bogofilter-0.92.2/src/paths.c	2004-05-25 03:53:36.000000000 +0000
+++ bogofilter-0.92.2.new/src/paths.c	2004-07-27 08:11:52.978408512 +0000
@@ -29,7 +29,7 @@
 #ifndef __riscos__
     { PR_ENV_BOGO, "BOGOFILTER_DIR", NULL },
     { PR_ENV_BOGO, "BOGODIR",	     NULL },
-    { PR_ENV_HOME, "HOME",	     BOGODIR }
+    { PR_ENV_HOME, "HOME_ETC",	     BOGODIR }
 #else
     { PR_ENV_HOME, "Choices$Write",  BOGODIR },
     { PR_ENV_HOME, "Bogofilter$Dir", NULL },
@@ -104,6 +104,7 @@
     size_t path_size, env_size;
 
     env = getenv(var);
+    if (env == NULL && strcmp(var, "HOME_ETC") == 0) env = getenv("HOME");
     if (env == NULL || *env == '\0') return NULL;
 
     env_size = strlen(env);
