diff -urN bogofilter-0.96.4.orig/configure.ac bogofilter-0.96.4/configure.ac
--- bogofilter-0.96.4.orig/configure.ac	2005-11-01 01:31:40.000000000 +0100
+++ bogofilter-0.96.4/configure.ac	2005-11-03 16:37:09.816592656 +0100
@@ -706,7 +706,7 @@
 export PATH
 
 need_my_gsl=1
-AC_ARG_WITH(included-gsl, AC_HELP_STRING([--with-included-gsl],[force use of shipped GSL rather than system GSL]),
+AC_ARG_WITH(included-gsl, ([--with-included-gsl],[force use of shipped GSL rather than system GSL]),
     if test "x$withval" = "xyes" ; then need_my_gsl=2 ; fi)
 
 if test "$need_my_gsl" = 1 ; then
