--- bogofilter-1.2.2/src/lexer.h~	2007-07-27 05:36:37.000000000 +0200
+++ bogofilter-1.2.2/src/lexer.h	2015-12-27 13:42:32.966722605 +0100
@@ -48,7 +48,7 @@
 typedef struct lexer_s {
     yylex_t  *yylex;
     char    **yytext;
-    int      *yyleng; /* DO NOT EVEN CONSIDER MAKING THIS SIZE_T! */
+    size_t   *yyleng; /* DO NOT EVEN CONSIDER MAKING THIS SIZE_T! */
 } lexer_t;
 
 extern lexer_t *lexer;
@@ -56,7 +56,7 @@
 
 /* in lexer_v3.l */
 extern token_t	yylex(void);
-extern int	yyleng;
+extern size_t	yyleng;
 extern char   * yytext;
 extern void	lexer_v3_init(FILE *fp);
 
